crates/sui-types/src/crypto.rs

```diff
@@ -396,6 +397,10 @@ impl SuiPublicKey for Secp256k1PublicKey {
     const SIGNATURE_SCHEME: SignatureScheme = SignatureScheme::Secp256k1;
 }
 
+impl SuiPublicKey for BLS12381PublicKey {
+    const SIGNATURE_SCHEME: SignatureScheme = SignatureScheme::Secp256k1;
```

Comment:
typo? SignatureScheme:: Secp256k1 

(make sure a test covers this too?)

Reply:
this is just to generate some tests. This will be deleted for the actual merge.

Reply:
Sorry, please ignore the changes in crypto.rs and messages_tests.rs, I only added this to generate some valid BLS signatures for the move tests.